### ISSUE TITLE
Not RL

### ISSUE THREAD

**Issue by itschenxi** [ZackAkil/deep-tic-tac-toe#4]
This is not reinforcement learning, this is supervised learning. 
hist = agent.model.fit(X, y, epochs=200, verbose=1, batch_size=64)

**Comment by ZackAkil** [comment]
you are correct that I'm not using a reinforcement learning framework, I've instead (as a learning exercise) have coded my own RL "play-observe-reward" loop  from scratch and using that to build a supervised learning dataset that can be used to train a model, how I think about it is within reinforcement learning there is supervised learning, the reinforcement learning bit is all in how it builds its own dataset by playing the game and recording which actions give the most reward, then the supervised learning bit is just learning those actions and maybe a pattern within them so that it can play in unseen scenarios
